### PR TITLE
Delete script

### PR DESCRIPTION
**Pull request by NMcKittie**
added a directory and script to delete out of date data #91

### CODE REVIEW

File: delete_old/delete.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    with conn.cursor() as cur:
+        cur.execute(sql_query)
+        conn.commit()
+
+
+def delete_weather_forecast(conn: connection) -> None:
+    """Delete any forecast that is out of date"""
+
+    sql_query = """DELETE FROM forecast
+                    WHERE (forecast_timestamp < CURRENT_TIMESTAMP) AND
```
Comment (DKWeeto) [inline delete_old/delete.py]: make sure not to delete most recent forecast (from up to 15 mins ago)

File: delete_old/delete.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    """Delete out of date flood warnings"""
+
+    sql_query = """DELETE FROM flood_warnings WHERE time_raised < CURRENT_TIMESTAMP - INTERVAL '7 days';"""
+
+    with conn.cursor() as cur:
+        cur.execute(sql_query)
+        conn.commit()
+
+
+def clear_the_data(config) -> None:
```
Comment (arj0713) [inline delete_old/delete.py]: Missing type hint

File: delete_old/delete.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    """delete all out of the out of date data"""
```
Comment (DKWeeto) [inline delete_old/delete.py]: capital letter :(

File: delete_old/delete.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        user=config["DB_USER"],
+        password=config["DB_PASSWORD"],
+        host=config["DB_HOST"],
+        port=config["DB_PORT"],
+        database=config["DB_NAME"]
+    )
+
+
+def delete_weather_alert(conn: connection) -> None:
+    """Delete any weather update that is out of date and has been notified"""
```
Comment (DKWeeto) [inline delete_old/delete.py]: No punctuation?

File: delete_old/delete.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+def clear_the_data(config) -> None:
+    """delete all out of the out of date data"""
+
+    with get_db_connection(config) as conn:
+        delete_weather_alert(conn)
+        delete_weather_forecast(conn)
+        delete_air_quality(conn)
+        delete_weather_reports(conn)
+        delete_flood_warnings(conn)
+
```
Comment (arj0713) [inline delete_old/delete.py:94]: Need to close the connection, leaving the context manager does not close it in `psycopg2`

File: delete_old/delete.py [hunk 6]
```diff
@@ -0,0 +1,98 @@
+"""deletes old out of date data from the database"""
```
Comment (DKWeeto) [inline delete_old/delete.py]: Capital letter and full stop? :0

**Review by DKWeeto — CHANGES_REQUESTED** [review]

File: delete_old/delete.py (hunk 6 above)
Comment (arj0713) [inline delete_old/delete.py]: Can we make this a proper sentence, also full stops across the board on doc strings

**Review by DKWeeto — APPROVED** [review]

**Pull request merged**